test/Worker.Extensions.Http.AspNetCore.Tests/Worker.Extensions.Http.AspNetCore.Tests.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I think no real changes were made here?

Reply:
yeah, initially had few changes but no changes now. This is keep coming up here even after fixing a couple times.